src/lib/util.js

```diff
@@ -158,7 +219,20 @@ export function get(url, ...rest) {
 
     return fetch(url, { headers })
         .then(checkStatus)
-        .then(parser);
+        .then(parser)
+        .catch((e) => {
+            // Make sure we are making a dl call
+            if (isNonDefaultDownloadHost(url)) {
+                // Retry download with default host.
+                setDownloadHostFallback();
+                return get(url, ...rest);
+            }
+
+            /* eslint-disable no-console */
+            console.error(e);
+            /* eslint-enable no-console */
+            return Promise.resolve();
```

Comment:
Not sure what we want to do here. We aren't really ever catching this call, so I'm just logging an error for now.  If it's not caught (e.g. by the preloader) preview will fail.